app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Enums\Status;
+use App\Http\Requests\Enum;
+use App\Models\User;
+use App\Models\Authentication;
```

Comment:
```suggestion
use App\Models\{
    User,
    Authentication
}
```
まとめられるものはまとめるようお願いします。

Reply:
承知しました。変更しました。